app/views/administrate/application/_collection.html.erb

```diff
@@ -18,13 +18,13 @@ to display a collection of resources in an HTML table.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<table class="collection-data">
+<table class="collection-data" aria-labelledby="page-title">
```

Comment:
@tysongach do you know if `aria-labelledby` translates to the same accessibility property as caption? If so, that's great. We might need to use `aria-describedby` depending on the mapping of the `caption` element.